src/routes/photos.py

```diff
@@ -0,0 +1,91 @@
+import datetime
+import os
+import pickle
+
+from typing import List
+from fastapi import APIRouter, Depends, status, UploadFile, File, HTTPException, Form
+from sqlalchemy.orm import Session
+import cloudinary
+import cloudinary.uploader
+
+from src.database.db import get_db
+from src.repository import photos as repository_photos
+from src.schemas import PhotoResponse, PhotoModel, TagsPhoto
+from src.conf.config import settings
+
+router = APIRouter(prefix="/photos", tags=["photos"])
+
+
+@router.post('/', response_model=PhotoResponse)
+async def create_photo(file: UploadFile = File(),
+                       description: str = Form(),
+                       db: Session = Depends(get_db)):
+    cloudinary.config(
+        cloud_name=settings.cloudinary_name,
+        api_key=settings.cloudinary_api_key,
+        api_secret=settings.cloudinary_api_secret,
+        secure=True
+    )
+    r = cloudinary.uploader.upload(file.file,
+                                   public_id=f'PhotoShare/{description[:7].replace(" ", "") +
+                                                           file.filename.replace(".", "")}',
+                                   overwrite=True)
+    url = cloudinary.CloudinaryImage(f'PhotoShare/{description[:7].replace(" ", "") +
+                                                   file.filename.replace(".", "")}') \
+        .build_url(crop='fill', version=r.get('version'))
+
+    return await repository_photos.create_photo(description, url, db)
+
+
+@router.get('/{photo_id}', response_model=PhotoResponse)
+async def read_photo(photo_id: int, db: Session = Depends(get_db)):
+    photo = await repository_photos.read_photo(photo_id, db)
+    if photo is None:
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="Photo not found")
+    return photo
+
+
+@router.put('/{photo_id}', response_model=PhotoResponse)
+async def update_photo(photo_id: int,
+                       file: UploadFile = File(),
+                       description: str = Form(),
+                       db: Session = Depends(get_db)):
+    cloudinary.config(
+        cloud_name=settings.cloudinary_name,
+        api_key=settings.cloudinary_api_key,
+        api_secret=settings.cloudinary_api_secret,
+        secure=True
+    )
+    r = cloudinary.uploader.upload(file.file,
+                                   public_id=f'PhotoShare/{description[:7].replace(" ", "") +
+                                                           file.filename.replace(".", "")}',
+                                   overwrite=True)
+    url = cloudinary.CloudinaryImage(f'PhotoShare/{description[:7].replace(" ", "") +
+                                                   file.filename.replace(".", "")}') \
+        .build_url(crop='fill', version=r.get('version'))
+
```

Comment:
Please, sanitize with variables
```suggestion
    # Create a clean file identifier
    clean_description = description[:7].replace(" ", "")
    clean_filename = file.filename.replace(".", "")
    public_id = f'PhotoShare/{clean_description}{clean_filename}'
    
    # Upload the file
    upload_response = cloudinary.uploader.upload(
        file.file,
        public_id=public_id,
        overwrite=True
    )
    
    # Generate the Cloudinary image URL
    url = cloudinary.CloudinaryImage(public_id)
```

Reply:
ok

Reply:
thank you, gut idea! done